core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -154,7 +154,7 @@ oppia.controller('ExplorationEditor', [
           successCallback();
         }
 
-        stateEditorTutorialFirstTimeService.init(
+        $scope.firstTime = stateEditorTutorialFirstTimeService.init(
```

Comment:
Oh, I meant just getting rid of the "$scope.firstTime =" bit, and leaving the rest. See the [diff](https://github.com/oppia/oppia/pull/1674/files) here.